### PR TITLE
a "publish" script to force push the "built" react app to the fs-webd…

### PR DESCRIPTION
**Pull request by joeycozza**
…ev/fresh-cra-template repo

### CODE REVIEW

File: packages/react-scripts/template/.travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 10
+  - '10'
```
Comment (redbugz) [inline packages/react-scripts/template/.travis.yml]: top travis uses double quotes, this is single. Sorry for the nitpick. I don't care which one wins.

File: freshCraTemplateUpdate.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+cd ${HOME}/tmp/my-test-app
+# The order of these sed commands is kind of important. The last one of uncommenting all lines needs to occur after
+# removing comments explaining in english steps for replacing the binding path
+sed -i.bak 's/{{BINDING_PATH}}/fresh-cra-template.frontier/' blueprint.yml
```
Comment (redbugz) [inline freshCraTemplateUpdate.sh]: I'm thinking @gmdayley might like to link to this as an example?
I wonder if we should make the path something like:
/frontier/fresh
/frontier/cra
/frontier/example
/frontier/sample

We can also decide later and change it easily

File: packages/react-scripts/template/.travis.yml (hunk 1 above)
Comment (joeycozza) [reply packages/react-scripts/template/.travis.yml]: yeah... my yaml lint fixer automatically put it to single quotes...

**Comment by joeycozza** [comment]
I'm going to merge this in to see where if any issues are. (I expect github push permissions to be a potential problem)

**Pull request merged**